apps/vmq_server/src/vmq_cluster_mon.erl

```diff
@@ -127,7 +127,7 @@ handle_info({nodeup, Node}, State) ->
     vmq_cluster:recheck(),
     {noreply, State};
 handle_info({gen_event_EXIT, vmq_cluster, _}, State) ->
-    plumtree_peer_service_events:add_sup_handler(vmq_cluster, [self()]),
```

Comment:
to pass `self()` as an arg to `add_sup_handler` was an actual bug which would result in a bad match error.

Reply:
This would be because the `vmq_cluster:init/1` only accepts the empty list `[]` as an argument, right? (not that versed in `gen_event` ... )